server.js

```diff
@@ -0,0 +1,29 @@
+/**/
+var express = require('express');
+var app = express();
+var http = require('http').Server(app);
+var io = require('socket.io')(http);
+
+app.use(express.static('public'));
+app.use('/js', express.static(__dirname + '/node_modules/bootstrap/dist/js')); // redirect bootstrap JS
+app.use('/js', express.static(__dirname + '/node_modules/jquery/dist')); // redirect JS jQuery
+app.use('/css', express.static(__dirname + '/node_modules/bootstrap/dist/css')); // redirect CSS bootstrap
+app.use('/css', express.static(__dirname + '/node_modules/animate.css')); // redirect CSS bootstrap
+
+app.get('/', function(req, res){
```

Comment:
You should move all Express and Socket.io to separated file and class (like `ExpressAppHandler`, `WebSocketHandler`).